lib/database/migrations/20231025101611-create-lhc-periods-view.js

```diff
@@ -1,14 +1,26 @@
 'use strict';
 
+const BASIC_PARTICLES_PROPERTIES = { // 2 * sqrt(ATOMIC_MASS / ATOMIC_NUMBER)
```

Comment:
Why basic?
Simply `PHYSICS_CONSTANTS` is enough for me. Or `PHYSICAL_CONSTANTS` as you named your table (not sure which one is correct, please check)

Reply:
refactored